src/index.js

```diff
@@ -75,9 +75,9 @@ function configureToMatchImageSnapshot({
 
       if (!pass) {
         const differencePercentage = result.diffRatio * 100;
-
+        const cwd = process.cwd();
         message = () => `Expected image to match or be a close match to snapshot but was ${differencePercentage}% different from snapshot (${result.diffPixelCount} differing pixels).\n`
-                  + `${chalk.bold.red('See diff for details:')} ${chalk.red(result.diffOutputPath)}`;
+                  + `${chalk.bold.red('See diff for details:')} ${chalk.red(result.diffOutputPath.replace(cwd, '.'))}`;
```

Comment:
This one is handy change which reduces diff output path length (which also allows using `. toThrowErrorMatchingSnapshot` in jest)

Reply:
I am not sure it is easier to find/read the path this way.

Maybe it is just me but I think that seeing:
`See diff for details /Users/username/path/to/__tests__/__image_snapshots__/diff.png`

is easier to read and more clear than:
`See diff for details ./__tests__/__image_snapshots__/diff.png`

Can you remove this change and just implement something like it at the test level? (in order to allow usage of `.toThrowErrorMatchingSnapshot()`

I think a change like this for the user experience should be its own PR where a discussion could happen.

Reply:
Haha, here's tricky part: if I add `NODE_ENV === 'test'` check it reduces coverage since it's always set to `test` in Jest env. I'll switch to regex option then for now.